src/app_engine/probers.py

```diff
@@ -104,20 +112,44 @@ def get(self):
 
 
 class ProbeColliderPage(webapp2.RequestHandler):
+  """Page to probe Collider instances."""
+
   def handle_collider_response(
       self, error_message, status_code, collider_instance):
```

Comment:
I had a hard time following what everything in this file is doing without really going line by line. Could you add a docstring to this function to describe what is happening?